### PR TITLE
Fix #14250

### LINKED ISSUE #14250: Implement missing ps*j
psj works, but pszj, pspj, ... does not

### PR DESCRIPTION
**Pull request by wargio**
Fixed #14250

`ps[puzwW+]j` are now available

### CODE REVIEW

**Comment by wargio** [comment]
https://github.com/radare/radare2/issues/14250

File: libr/core/cmd_print.c [hunk 1]
```diff
@@ -4507,6 +4507,49 @@ R_API void r_print_code(RPrint *p, ut64 addr, const ut8 *buf, int len, char lang
 	}
 }
 
+static void get_string_section(RCore *core, const char** section_name, ut64* address) {
```
Comment (radare) [inline libr/core/cmd_print.c]: wtf we have 3 functions that do the same. see r_core_get_section_name

Comment (wargio) [reply libr/core/cmd_print.c]: Oh. I just refactored. I'll remove it and use that one you suggested

File: libr/core/cmd_print.c [hunk 2]
```diff
@@ -4507,30 +4507,19 @@ R_API void r_print_code(RPrint *p, ut64 addr, const ut8 *buf, int len, char lang
...
 	char *str;
-
-	get_string_section (core, &section_name, NULL);
+	const char* section_name = r_core_get_section_name (core, core->offset);
+	if (!section_name) {
+		section_name = "unknown";
+	} else {
+		// cleaning useless spaces in section name in json data.
+		char* p = (char*) section_name;
+		for (; *p == ' '; p++) {}
```
Comment (radare) [inline libr/core/cmd_print.c]: Use rstrtrimro

**Comment by radare** [comment]
there are 3 xx

**Comment by radare** [comment]
thanks!!

**Pull request merged**